README.md

```diff
@@ -1,8 +1,48 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, Javascript/Typescript, React, Node, Express, HTML/CSS, Python
+
+**Competent:** Scala, MongoDB, C/C++, Springboot, Docker, Github Actions, Mocha, JUnit, Swing, Atom
+
+**Familiar:** C#, Hadoop, Objective-C, Swift, pandas, pygame
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_KaamilParmar](https://github.com/jarviscanada/jarvis_data_eng_KaamilParmar)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KaamilParmar/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_KaamilParmar/tree/main/linux_sql)]: Implemented a Bash-based monitoring agent to consistently track and store hardware specs and usage data of host machines. Deployed the application on Docker, syncing data with a PostgreSQL database. Conducted operations on a CentOS virtual machine hosted on Google Cloud Platform. Automated data collection using crontab for regular updates. Tested the system functionality through command-line interface (CLI) for robust performance.
+
+**SQL Queries** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_KaamilParmar/tree/masterhttps://github.com/jarviscanada/jarvis_data_eng_KaamilParmar/tree/main/SQL)]: Implemented and executed SQL queries on PostgreSQL tables within a Docker container. Utilized DBeaver for query execution, focusing on exercises from pgexercise. Demonstrated proficiency in CRUD operations, Joins, Aggregate Functions, and advanced SQL techniques. Emphasized database setup and schema understanding for effective query execution.
+
+
+## Highlighted Projects
+**E-Commerce Application** [[GitHub](https://github.com/klparmar/react-ecomm-stripe-proj)]: Deployed a responsive, dynamic, and mobile-adaptive FullStack web application for E-commerce, utilizing Javascript, Java Spring and Bootstrap as a platform for businesses to sell products
+
+**Retro Video Games in Java** [[GitHub](https://github.com/klparmar/Java-Projects)]: Created various interactive gaming applications utilizing numerous Java libraries such as Atari Breakout, Space Invaders, Frogger, and Pong.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2023-present)**: Developed and deployed several applications with languages and frameworks such as Bash, SQL, Java, Spark, Scala, Python, Javascript, React, Spring, Docker and worked in an Agile and Scrum environment. Utilized Git and GitFlow for version control
+
+**Full Stack Developer, InspoBoard (2022-2023)**: Designed, developed, and maintained a SaaS analytics application for ad agencies and businesses to store, organize, and manage Facebook and TikTok ads using the MERN (MongoDB, Express.js, React, Node.js) stack.
```

Comment:
Is it possible to expand these sections at all? They feel very sparse. Some additional details would leave a bigger impact.